src/views/repos/show.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import GraphEditor from '../../components/graph_editor/flux/components/graph_editor_base'
```

Comment:
Line is too long.<br>'import' is only available in ES6 (use esnext option).<br>Missing semicolon.